src/test/resources/features/list.feature

```diff
@@ -1,29 +1,33 @@
 Feature: List actions
 
-  @ArchiveList
+  @CreateList
   Scenario: New list creation
     Given I log in as admin user
-    When I select TestBoard board
+    When I select BoardForList board
     And I create a new list with "ListTest" as title
     Then I should see the new created list with the given title
 
   @MoveAllCards
   Scenario: Move all cards in list
     Given I am logged in as admin user
-    When I select TestBoard board
-    And I move all cards in Tasks1 to Tasks2
+    And I select MoveSortBoard board
+    And there are following lists in the board
+      | SourceList |
+      | TargetList |
```

Comment:
Justify this line please.

Reply:
The given lists are going to be saved in the context, so the their data can be used for assertions and hooks.